code_generator/GdkCodeGenerator/Templates/UnityComponentConversionGenerator.tt

```diff
@@ -287,15 +310,91 @@ namespace <#= qualifiedNamespace #>
                 );
             }
 
-<# foreach (var command in commandDetailsList) { #>
-            private void On<#= command.CommandName #>Request(CommandRequestOp op)
+<# foreach (var commandDetails in commandDetailsList) {
+    var wrappedCommandRequestType = $"{qualifiedNamespace}.{componentDetails.ComponentName}.{commandDetails.CommandName}.ReceivedRequest";
+    var commandRequestBufferType = $"{qualifiedNamespace}.{componentDetails.ComponentName}.CommandRequests.{commandDetails.CommandName}";
+
+    var wrappedCommandResponseType = $"{qualifiedNamespace}.{componentDetails.ComponentName}.{commandDetails.CommandName}.ReceivedResponse";
+    var commandResponseBufferType = $"{qualifiedNamespace}.{componentDetails.ComponentName}.CommandResponses.{commandDetails.CommandName}";
+
+    var commandStorage = $"{commandDetails.CamelCaseName}Storage";
+#>
+            private void On<#= commandDetails.CommandName #>Request(CommandRequestOp op)
             {
-                // TODO: Implement commands
+                if (!IsValidEntityId(op.EntityId, "CommandRequestOp", out var entity))
+                {
+                    return;
```

Comment:
should we add some kind of log message or is this already done by `IsValidEntityId`?

Reply:
The latter :)